plugins/org.eclipse.n4js/src/org/eclipse/n4js/validation/N4JSJavaScriptVariantHelper.java

```diff
@@ -11,18 +11,19 @@
 package org.eclipse.n4js.validation;
 
 import org.eclipse.emf.ecore.EObject;
+import org.eclipse.n4js.N4JSGlobals;
 
 import com.google.inject.Singleton;
 
-import org.eclipse.n4js.N4JSGlobals;
-
 /**
  * This class defines the constraints to be checked N4JS and N4JSD.
  *
  */
 @Singleton
 public class N4JSJavaScriptVariantHelper extends BaseJavaScriptVariantHelper {
```

Comment:
Not sure, but could maybe this class be simplified as well, now that N4JSXJavaScriptVariantHelper is removed?

Reply:
Not that easy. We have `N4JSJavaScriptVariantHelper extends BaseJavaScriptVariantHelper implements JavaScriptVariantHelper`. Whole that structure has to be refactored, but as part of anticipated redesign of language variants.